CRM/Report/Form/Contribute/DeferredRevenue.php

```diff
@@ -362,7 +362,7 @@ public function limit($rowCount = NULL) {
   public function where() {
     parent::where();
     $startDate = date('Y-m-01');
-    $endDate = date('Y-m-t', strtotime(date('ymd') . '+11 month'));
+    $endDate = (new DateTime('+11 month'))->format('Y-m-d');
```

Comment:
This changes the format string from `'Y-m-t'` to `'Y-m-d'`. That is, from the last day of the month to the current day of the month. Is that intentional?

Reply:
No, good catch thank you @Sjord